### PR TITLE
Adding capability about new chunking - refs #25682 (#26368)

### PR DESCRIPTION
**Pull request by rullzer**
CC: @nickvergessen @MorrisJobke @LukasReschke @icewind1991

### CODE REVIEW

**Comment by MorrisJobke** [comment]
owncloud/core#25682

**Comment by MorrisJobke** [comment]
Tested and works 👍

File: apps/dav/lib/Capabilities.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+
+namespace OCA\DAV;
+
+use OCP\Capabilities\ICapability;
+
+class Capabilities implements ICapability {
+
+	public function getCapabilities() {
```
Comment (LukasReschke) [inline apps/dav/lib/Capabilities.php]: Will add a small test for that.

**Review by LukasReschke — CHANGES_REQUESTED** [review]

**Review by LukasReschke — APPROVED** [review]

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**